IdentityAccess/src/Kreta/IdentityAccess/Infrastructure/Symfony/HttpAction/UsersByIdsAction.php

```diff
@@ -30,13 +31,26 @@ public function __construct(QueryBus $queryBus)
 
     public function __invoke(Request $request) : JsonResponse
     {
-        $ids = explode(',', $request->query->get('ids'));
+
```

Comment:
Instead of using this ugly conditional you can decouple this action into two different actions